cmd/clusterctl/clusterdeployer/bootstrap/kind/kind.go

```diff
@@ -31,6 +31,11 @@ const (
 	kindClusterName  = "clusterapi"
 )
 
+var (
+	// ignoredOptions lists the options not supported by delete and kubeconfig-path.
+	ignoredOptions = []string{"config", "image", "retain"}
```

Comment:
i think `createSpecificOptions` would be a better name here.

Reply:
imho with the helper, this name brings more confusion than clarification.